Automattic/jetpack
# Update upgrade nudge text in editor

This small change update the text in the editor, so it makes more sense.

#### Jetpack product discussion



1203313664893883-as-1203337382116441

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






Go to Gutenberg without a Social plan, and check the upgrade nudge. Should have the new text:
`You have [x] shares remaining. [Upgrade now](link) to share more.  `

<img width="280" alt="CleanShot 2022-11-18 at 11 02 28@2x" src="https://user-images.githubusercontent.com/36671565/202678164-5f6346a1-e64b-422f-9c51-19a3d1d235bc.png">
